intrinsic_cal/CMakeLists.txt

```diff
@@ -95,7 +95,7 @@ set(CMAKE_CXX_FLAGS ${CMAKE_CSS_FLAGS} "-fPIC")
 ## CATKIN_DEPENDS: catkin_packages dependent projects also need
 ## DEPENDS: system dependencies of this project that dependent projects also need
 catkin_package(
-  INCLUDE_DIRS include /usr/include/eigen3
+  INCLUDE_DIRS /usr/include/eigen3
```

Comment:
why is this (exported) include path hard coded?

That is really not something we'd want to do. If anything, it should probably be something like `${eigen_INCLUDE_DIRS}` (or whatever the actual name of the variable is).